xds/src/test/java/io/grpc/xds/XdsClientImplTest.java

```diff
@@ -336,7 +336,7 @@ public void ldsResponseWithoutMatchingResource() {
     // Client sends an LDS request for the host name (with port) to management server.
     verify(requestObserver)
         .onNext(eq(buildDiscoveryRequest(NODE, "", TARGET_AUTHORITY,
-            XdsClientImpl.ADS_TYPE_URL_LDS, "")));
+            XdsClientImpl.ADS_TYPE_URL_LDS_v2, "")));
```

Comment:
A high level comment about tests. So by renaming `ADS_TYPE_URL_LDS` to `ADS_TYPE_URL_LDS_v2` we are only testing/validating v2 responses. What about testing v3 responses? Is there a separate PR for it or a TODO comment to indicate it needs to be done?

Reply:
Currently we are still sending v2 request, so responses are expected to be v2. Testing v3 response will be after v3 bootstrap and env flag enabled.

Reply:
Yes, I realized it later. Sounds good.